client/form.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <!-- <script defer src="./assets/bundle.js"></script> -->
+    <script defer src="assets/js/form.js"></script>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="assets/css/form.css" />
+    <link rel="stylesheet" href="assets/css/style.css" />
+    <title>Journal Entry</title>
+</head>
+<body>
+    <nav>
+        <h1>JOURNEL</h1>
+        <div>
+            <a href="#">Anime</a>
+            <a href="#">Food</a>
+            <a href="#">Movies</a>
+        </div>	
+    </nav>
+    <form class="journal-post-form" action="./" method="POST">
+        <div class="entry-title">Create a journal entry</div>
+        <div class="form-item">
+          <label for="category-selector" class="form-label">Select Your Post Category</label>
+            <input id="category-selector" list="categories" name="Category">
+            <datalist id="categories">
+                <option value="Anime">
+                <option value="Food">
+                <option value="Movies">
+            </datalist>
+            <!-- <span class="form-error">No category selected</span> -->
```

Comment:
Safe to remove this comment. Look into form input attributes for data validation - potentially the pattern attribute.
required attribute will work here.